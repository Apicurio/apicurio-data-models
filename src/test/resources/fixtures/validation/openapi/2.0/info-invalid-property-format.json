{
  "swagger": "2.0",
  "info":{
    "description":"This is a sample server Petstore server via JSON.",
    "version":"1.0.0",
    "title":"Swagger Petstore (JSON)",
    "termsOfService":"http://helloreverb.com/terms/",
    "contact":{
      "email":"invalid_email"
    },
    "license":{
      "name":"Apache 2.0",
      "url":"invalid_url"
    }
  },
  "paths": {
  }
}