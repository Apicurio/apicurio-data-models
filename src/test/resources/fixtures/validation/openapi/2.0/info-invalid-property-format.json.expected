[CTC-002] |medium| {/info/contact->email} :: Contact Email is an incorrect format.
[LIC-002] |medium| {/info/license->url} :: License URL is an incorrect format.